### ISSUE TITLE
Correct Spelling Issue with aria-descrivedby="not-eaten"

### ISSUE THREAD

**Issue by jfc3** [jfc3/atehere#937]
Need to correct Spelling Issue with aria-descrivedby="not-eaten". It should be aria-descrbedby="not-eaten" instead of aria-descrivedby="not-eaten".

**Comment by jfc3** [comment]
Removed aria-descrivedby related code and fixed if statement to add tags inside the echo in the INDEX.php file, NOTEATEN.php, and TEST.php.

Updating ReadMe file with these changes.
